lyse/__init__.py

```diff
@@ -724,15 +735,23 @@ def __init__(self, h5_path, run_paths, no_write=False):
 
         super().__init__(h5_path, no_write=no_write)
 
-        if isinstance(run_paths, pandas.DataFrame):
-            run_paths = run_paths['filepath']      
-        self.runs = {path: Run(path,no_write=True) for path in run_paths}
+        if isinstance(shot_paths, pandas.DataFrame):
+            shot_paths = shot_paths['filepath']      
+        self.__shots = {path: Shot(path,no_write=True) for path in shot_paths}
+        
+    @property
+    def runs(self):
+        warnings.warn("The 'runs' attribute has been renamed to 'shots'. 'runs' will be removed in lyse v4.0. Please update your analysis code to use the 'shots` attribute", DeprecationWarning)
+        return copy.copy(self.__shots)
+
+    def shots(self):
+        return copy.copy(self.__shots)
```

Comment:
I believe this needs a property decorator

```suggestion
    @property
    def shots(self):
        return copy.copy(self.__shots)
```

Reply:
Fixed in 016768d